fstest/fstests/fstests.go

```diff
@@ -471,6 +471,51 @@ func TestFsMove(t *testing.T) {
 	// 2: hello sausage?/../z.txt
 }
 
+// Copy src to this remote using server side move operations.
+//
+// Will only be called if src.Fs().Name() == f.Name()
+//
+// If it isn't possible then return fs.ErrorCantDirCopy
+//
+// If destination exists then return fs.ErrorDirExists
+
+// TestFsDirCopy tests DirCopy
+func TestFsDirCopy(t *testing.T) {
+	skipIfNotOk(t)
+
+	// Check have DirCopy
+	doDirCopy := remote.Features().DirCopy
+	if doDirCopy == nil {
+		t.Skip("FS has no DirCopier interface")
+	}
+
+	// Check it can't copy onto itself
+	err := doDirCopy(remote)
+	require.Equal(t, fs.ErrorDirExists, err)
+
+	// new remote
+	newRemote, _, removeNewRemote, err := fstest.RandomRemote(RemoteName, false)
+	require.NoError(t, err)
+	defer removeNewRemote()
+
+	// try the move
+	err = newRemote.Features().DirCopy(remote)
+	require.NoError(t, err)
+
+	// check remotes
+	// FIXME: Prints errors.
+	fstest.CheckListing(t, remote, []fstest.Item{})
```

Comment:
You don't need the []fstest.Item{} you can just write this.  

I think you are expecting a copy, so you would expect

```
fstest.CheckListing(t, remote, file1, file2)
fstest.CheckListing(t, newRemote, file1, file2)
```

Wouldn't you?
Though you don't appear to have created any files?